### DISCUSSION TITLE
Use Linux System user name and password

### DISCUSSION THREAD

**Discussion by Raidm404** [prasathmani/tinyfilemanager#1105] [Ideas]
Hello,
 Could some one point me how I can use a standard Linux  system user with password to do the login process and set the root as the home directory of the system user.

Any help will be appreciated

**Comment by prasathmani** [comment]
TFM will not support standard system user login. you can customise to use LDAP login https://i12bretro.github.io/tutorials/0046.html

**Reply by Raidm404** [reply]
I am really looking to do a customization on this project. I am working on a hosting control panel and  I love to use this as the filemanager. The problem is the control panel is written on python .  But I can place the script on a php docroot that runs on nginx  server.  If some one help me to implement these changes ,  I can use it along with a porject and it will be a good reference too.

If some one can take this as a paid work  I can afford that too lol :)

**Comment by mbucas** [comment]
PAM is a part of the authentication stack on Linux (and maybe other systems).

I found this possible path to something close to what you want to do : https://stackoverflow.com/questions/1667996/is-it-possible-to-have-a-php-script-authenticate-users-with-their-linux-user-inf

However, I don't know what should be changed in PAM configuration (a broken PAM makes you unable to login into a system...) neither do I fully understand the security consequences.

Good luck !
